Stripe/StripeiOSTests/CustomerAdapterTests.swift

```diff
@@ -126,29 +126,7 @@ class CustomerAdapterTests: APIStubbedTestCase {
 // TODO: Uncomment when enabling USBankAccount
 /*
     func testFetchPM_CardAndUSBankAccount() async throws {
-        let expectedPaymentMethods_card = [STPFixtures.paymentMethod()]
-        let expectedPaymentMethods_cardJSON = [STPFixtures.paymentMethodJSON()]
-
-        let expectedPaymentMethods_usbank = [STPFixtures.bankAccountPaymentMethod()]
-        let expectedPaymentMethods_usbankJSON = [STPFixtures.bankAccountPaymentMethodJSON()]
-
-        let apiClient = stubbedAPIClient()
-        // Expect 1 call per PM: cards
-        stubListPaymentMethods(key: exampleKey, paymentMethodType: "card", paymentMethodJSONs: expectedPaymentMethods_cardJSON, expectedCount: 1, apiClient: apiClient)
-        stubListPaymentMethods(key: exampleKey, paymentMethodType: "us_bank_account", paymentMethodJSONs: expectedPaymentMethods_usbankJSON, expectedCount: 1, apiClient: apiClient)
-
-        let ekm = MockEphemeralKeyEndpoint(exampleKey)
-        var configuration = configuration()
-        configuration.paymentMethodTypes = [.card, .USBankAccount]
-        let sut = StripeCustomerAdapter(customerEphemeralKeyProvider: ekm.getEphemeralKey,
-                                        setupIntentClientSecretProvider: { return "si_" },
-                                        configuration: configuration, apiClient: apiClient)
-        let pms = try await sut.fetchPaymentMethods()
-
-        XCTAssertEqual(pms.count, 2)
-        XCTAssertEqual(pms[0].stripeId, expectedPaymentMethods_card[0].stripeId)
-        XCTAssertEqual(pms[1].stripeId, expectedPaymentMethods_usbank[0].stripeId)
-        await waitForExpectations(timeout: 2)
+    // TODO
```

Comment:
?

Reply:
I feel like the code is going to change quite a bit when we go to enable usbankaccount,  so I just decided to nuke the test and re-write it when we go to enable usbankaccount.